examples/Makefile.am

```diff
@@ -83,18 +82,20 @@ VPP_INPUT_CFLAGS = \
 
 if ENABLE_X11
 simpleplayer_LDADD = $(YAMI_DECODE_LIBS)
-simpleplayer_LDFLAGS = $(LIBYAMI_CFLAGS)
+simpleplayer_CPPFLAGS = $(YAMI_COMMON_CFLAGS) $(AM_CPPFLAGS)
+simpleplayer_LDFLAGS = $(AM_LDFLAGS)
```

Comment:
This is unnecessary.  If AM_LDFLAGS is the only thing in the per-target override, then just don't define the override (e.g. don't define `simpleplayer_LDFLAGS`).  That is, if the per-target override is undefined, then AM_LDFLAGS is implied implicitly to the target.  Same comment applies for other targets.